google/cloud/pubsublite/internal/wire/admin_client_impl.py

```diff
@@ -88,12 +89,12 @@ def create_subscription(
     ) -> Subscription:
         path = SubscriptionPath.parse(subscription.name)
         return self._underlying.create_subscription(
-            request={
-                "parent": str(path.to_location_path()),
-                "subscription": subscription,
-                "subscription_id": path.name,
-                "skip_backlog": (starting_offset == BacklogLocation.END),
-            }
+            request=CreateSubscriptionRequest(
```

Comment:
This is potentially a breaking behavior change due to some known issues in proto-plus if you have any users that have passed in a subscription as a dictionary

https://github.com/googleapis/proto-plus-python/issues/161 comes to mind, but I think there was another one specifically about mixing dictionaries and protos, but I'm having trouble finding it.

Reply:
Found it! https://github.com/googleapis/python-tasks/issues/148

Reply:
Acknowledged. I'm okay with this- the type hints tell the user the exact type they need to pass. I don't think this library has wide enough usage that its expected to be bug-compatable with past versions: this is only a breaking change if proto-plus-python has a bug preventing usage of a dict where it should be usable, and the user is ignoring the type hint provided on the method.

Reply:
I haven't looked closely enough to know whether it's a breaking change or not, but if it is, we should update the issue title to not use `chore` accordingly.

Reply:
Done.